src/panels/config/script/ha-script-picker.ts

```diff
@@ -68,10 +69,25 @@ class HaScriptPicker extends LitElement {
                         <div>${computeStateName(script)}</div>
                       </paper-item-body>
                       <div class="actions">
-                        <a href=${`/config/script/edit/${script.entity_id}`}>
+                        <a
+                          href=${ifDefined(
+                            script.attributes.id
+                              ? `/config/script/edit/${script.attributes.id}`
+                              : undefined
+                          )}
+                        >
                           <paper-icon-button
                             icon="hass:pencil"
+                            .disabled=${!script.attributes.id}
                           ></paper-icon-button>
+                          ${!script.attributes.id
+                            ? html`
+                                <paper-tooltip position="left">
```

Comment:
You need to import `paper-tooltip`
`import "@polymer/paper-tooltip/paper-tooltip";`